src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <main class="page__content">
+      <div class="calendar calendar--month-length--29">
+        <div class="calendar__day calendar__day--start-day--sun"></div>
```

Comment:
Maybe you can use the modifier in the main block instead of the child block.

Reply:
And you need to add 2 blocks. `Create a markup for the calendar block with 31 days inside`

Reply:
yee, I have 31, but modificator makes 30,31s hidden, if I put mont-length--31 it should show 31 block. or do you mean somth else?

Reply:
> yee, I have 31, but modificator makes 30,31s hidden, if I put mont-length--31 it should show 31 block. or do you mean somth else?

Yeah, I mean this. So you can change the modifier so that, I can see 31 days when I open the demo link.

Reply:
> Maybe you can use the modifier in the main block instead of the child block.


Yes, you should do it

Reply:
Got you guys, thanks! done!